server/routerlicious/packages/lambdas/src/deli/lambdaFactory.ts

```diff
@@ -122,17 +127,22 @@ export class DeliLambdaFactory extends EventEmitter implements IPartitionLambdaF
         // both to be safe. Empty sring denotes a cache that was cleared due to a service summary or the document
         // was created within a different tenant.
         if (document.deli === undefined || document.deli === null) {
-            context.log?.info(`New document. Setting empty deli checkpoint`, { messageMetaData });
+            const message = "New document. Setting empty deli checkpoint";
+            context.log?.info(message, { messageMetaData });
+            Lumberjack.error(message, { messageMetaData });
             lastCheckpoint = getDefaultCheckpooint(leaderEpoch);
         } else {
             if (document.deli === "") {
-                context.log?.info(`Existing document. Fetching checkpoint from summary`, { messageMetaData });
+                const docExistsMessge = "Existing document. Fetching checkpoint from summary";
+                context.log?.info(docExistsMessge, { messageMetaData });
+                Lumberjack.error(docExistsMessge, { messageMetaData });
 
                 const lastCheckpointFromSummary =
                     await this.loadStateFromSummary(tenantId, documentId, gitManager, context.log);
                 if (lastCheckpointFromSummary === undefined) {
                     const errMsg = "Could not load state from summary";
                     context.log?.error(errMsg, { messageMetaData });
+                    Lumberjack.error(errMsg, { messageMetaData });
```

Comment:
Let's use `getLumberBaseProperties()` instead of `{ messageMetaData }`, here and elsewhere.